client/extensions/woocommerce/app/dashboard/required-plugins-install-view.js

```diff
@@ -318,8 +319,45 @@ class RequiredPluginsInstallView extends Component {
 		return 100;
 	}
 
+	startSetup = () => {
+		this.setState( {
+			engineState: 'INITIALIZING',
+		} );
+	}
+
+	renderConfirmScreen = () => {
+		const { translate } = this.props;
+		return (
+			<div className="card dashboard__setup-wrapper dashboard__setup-confirm">
+				<SetupHeader
+					imageSource={ '/calypso/images/extensions/woocommerce/woocommerce-setup.svg' }
+					imageWidth={ 160 }
+					title={ translate( 'Have something to sell?' ) }
+					subtitle={ translate(
+						'If you\'re located in the {{strong}}United States{{/strong}} ' +
+						'or {{strong}}Canada{{/strong}}, you can now add a store to sell your physical ' +
+						'products right on your site!',
+						{
+							components: { strong: <strong /> }
```

Comment:
Due to this component, the translate function returns an array, which is flagging a proptypes warning for  `subtitle` in `SetupHeader`. Should be fixed if you accept both string and array as types for `subtitle`.